### DISCUSSION TITLE
Welcome to B660M-HDV---i5-1200F Discussions!

### DISCUSSION THREAD

**Discussion by dclive** [dclive/B660M-HDV-i5-1200F-Hackintosh#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OceanApart** [comment]
@dclive Did you overclock 12400F? I got only 1612 single-core score and 6062 multi-core score with another B660M Motherboard and modified EFI base on yours.

**Comment by dclive** [comment]
I did not overclock.  

I can’t say how this would work on a non-B660M HDV.  That’s all I used for testing this EFI.

**Comment by dclive** [comment]
What is your RAM running at? 
Do you have any wattage limits in place in the BIOS?

**Reply by OceanApart** [reply]
2 * 8GB 2400 MHz DDR4 RAM, but i don't think it makes so much difference. I've ASUS Performance Enhancement (APE) 3.0 enabled, so there is no wattage litmits I guess. And I can see 75W on Intel Power Gadget.app when running Cinebench R23 Multi-core test, 11000+ scores in the end. I wonder how much wattage in max do you get?

<img width="1432" alt="image" src="https://github.com/dclive/B660M-HDV-i5-12400F-Hackintosh/assets/19257940/f567cba0-7980-4560-93a4-62854aa40670">

**Reply by OceanApart** [reply]
Another strange thing is that I found three repo with 2000+ scores on single-core on Github which tests on 12600K model.

- https://github.com/yzchan/MSI-MAG-B660M-MORTAR-DDR4-12600K-EFI
- https://github.com/Yash-Garg/OpenCore-B660M-12600K
- https://github.com/echokk11/Hackintosh_i7-12700kf_ASUS-B660M-PLUS-D4_5700xt

**Comment by dclive** [comment]
I suggest you retest with modern RAM.  You can see in my results that when RAM speed increased, everything else did too, particularly multicore speeds.  

75W for a 12600K is very low.  You are wattage limited (in addition to RAM speed limited).  

https://www.tomshardware.com/reviews/intel-core-i9-12900k-and-core-i5-12600k-review-retaking-the-gaming-crown/3 shows 120W to me.  If that's right (and I have no reason to doubt THW) then you're only using a bit over half your CPU's capability.

**Reply by OceanApart** [reply]
Thx for replying.
To be clear, my cpu is **i5-12400F**. What I supposed to say is that 2000+ scores single-core on **12600KF** sounds reasonable to me. So did your cpu reach over 75W on Hackintoh?
And My RAM is low-end product, 2400Mhz is the maximum. I'm not counting on that.

**Comment by dclive** [comment]
OK.  There's enough different there that it is hard for me to comment much.  All I can say is those are the scores I got, with the BIOS and RAM versions / types that I used.  If you're using a different MB, my suggestion would be to test a few settings on the board and see what makes a difference, and see if you can borrow PC3200 or PC3600 RAM for a few minutes to test again.  

The machine is now dead due to a power issue, so I cannot test at all anymore, alas.  I have a Gigabyte Elite B660 that's pretty similar, but even that's no longer used as I've replaced it with Apple Silicon - the MBP14" M3P called out to me one day.
